CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 The changelog for `IGListKit`. Also see the [releases](https://github.com/instagram/IGListKit/releases) on GitHub.
 
+2.0.0
+-----
+
+### Enhancements
```

Comment:
Can you rebase and put this all under "Master"? I'm going to change the version in another diff